context/app/static/js/components/detailPage/visualization/VisualizationTracker/utils.ts

```diff
@@ -109,7 +109,7 @@ export function fireVitessceEvent(ref: MutableRefObject<VitessceInteraction>) {
     trackEvent({
       category: 'Visualization',
       action: 'Interaction',
-      value: stringifyVitessceInteraction(ref.current),
+      label: stringifyVitessceInteraction(ref.current),
```

Comment:
I'm not sure this will work well for these events since they spit out long digests of activity (i.e. this wouldn't be categorizable by label)

Reply:
Do you have a suggestion for how we can track this? If it's not a number `value` won't work either.

Reply:
Should we file a ticket to revisit this?

Reply:
Filed https://hms-dbmi.atlassian.net/browse/CAT-546